docs/helm/docker-desktop-deployment.md

```diff
@@ -127,37 +127,28 @@ and therefore requires a large amount of resources out-of-the-box. To reduce the
 size of the deployment so it can run on a single machine we'll need to reduce
 the number of pods deployed and the memory requirements for several others.
 
-Fortunately this can all be achieved with one, albeit large, command as shown below:
+To install the Enterprise on localhost we need to use the [local-dev-values.yaml](../../docs/helm/values/). Once downloaded execute the command below to deploy.
 
 ```bash
 helm install acs alfresco/alfresco-content-services \
-  --set externalPort="80" \
-  --set externalProtocol="http" \
-  --set externalHost="localhost" \
+  --values= local-dev-values.yaml \
   --set global.tracking.sharedsecret=$(openssl rand -hex 24) \
-  --set global.alfrescoRegistryPullSecrets=quay-registry-secret \
-  --set repository.replicaCount=1 \
-  --set transformrouter.replicaCount=1 \
-  --set pdfrenderer.replicaCount=1 \
-  --set imagemagick.replicaCount=1 \
-  --set libreoffice.replicaCount=1 \
-  --set tika.replicaCount=1 \
-  --set transformmisc.replicaCount=1 \
-  --set postgresql-syncservice.resources.requests.memory="500Mi" \
-  --set postgresql-syncservice.resources.limits.memory="500Mi" \
-  --set postgresql.resources.requests.memory="500Mi" \
-  --set postgresql.resources.limits.memory="500Mi" \
-  --set alfresco-search.resources.requests.memory="1000Mi" \
-  --set alfresco-search.resources.limits.memory="1000Mi" \
-  --set share.resources.limits.memory="1500Mi" \
-  --set share.resources.requests.memory="1500Mi" \
-  --set repository.resources.limits.memory="2500Mi" \
-  --set repository.resources.requests.memory="2500Mi" \
   --atomic \
   --timeout 10m0s \
   --namespace alfresco
 ```
 
+#### Search Enterprise deployment
+
+To install the Search Enterprise we need to use the [local-dev-values.yaml & enterprise-search-values.yaml](../../docs/helm/values/). Once downloaded execute the command below to deploy.
+
+```bash
+helm install acs alfresco/alfresco-content-services \
+  --values= local-dev-values.yaml \
+  --values= enterprise-search-values.yaml \
+  --namespace alfresco
+```
+
 > NOTE: The command will wait until the deployment is ready so please be patient. See below for [troubleshooting](./docker-desktop-deployment.md#troubleshooting) tips.
 
 The command above installs the latest version of ACS Enterprise. To deploy a previous version of ACS Enterprise follow the steps below.
```

Comment:
you need to update/simplify also the next section

I would add a dedicated section for this